examples/default.rs

```diff
@@ -2,11 +2,10 @@ extern crate cursive;
 extern crate cursive_hexview;
 
 use cursive::views::{Dialog, DummyView, LinearLayout, TextView};
-use cursive::Cursive;
 use cursive_hexview::{DisplayState, HexView};
 
 fn main() {
-    let mut cur = Cursive::default();
+    let mut cur = cursive::default();
```

Comment:
Same here!

Reply:
I needed this while working on these changes and it wasn't yet merged to your master, so I don't know how to do it differently or why it ended up in the pull request even though I created two different branches for it. I'll wait till you let me know how you want everything else, I'm not confident I won't make a mess if I rebase.

Reply:
It ended there, because it was in your branch ;) to get rid of it, you need to rebase your changes and delete that commit. Afterwards force push the changes. More info about rebasing can be found here: https://git-scm.com/book/en/v2/Git-Branching-Rebasing

Reply:
Thanks for the link. I'm quite experienced with git and rebasing, it's just the whole github and pull requests thing that's very alien to me. I guess it's a matter of getting used to it and finding the few tricks that make it work. For example when you merged the other pull request, I updated my master with your master and now it says that mine is both one ahead and one behind!! go figure. I did find (this link)[https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request] which seems to go into the rebasing for pull requests. Wish me luck!